code/hll.pins

```diff
@@ -1,4 +1,4 @@
 dns-client = "7.0.3"
-js_of_ocaml = "5.7.2"
+js_of_ocaml = "5.9.0"
```

Comment:
Is this change intended for this ticket?

Reply:
Nice catch, it was inteded for me to compile on 5.3.0, but it will break the Dockerfile-based build